challenge_3/python/ekstrah/src/a.py

```diff
@@ -0,0 +1,18 @@
+def findingMajoList(aList): #{
+    bList = set(aList)
+    bList = list(bList)
+    for i in range(0, len(bList)): #{
```

Comment:
A major tool in python is the ability to iterate over most objects.
So instead of iterating an index and then looking up the item list you can instead iterate over the list. 

Instead of 
````Python
for i in range(0,X):
    print(bList[i])
````

you can just do 

````python
for item in bList:
    print(item)
````
Check out this [video](https://www.youtube.com/watch?v=EnSu9hHGq5o)
if you are interested in learning more about this topic (iterators) and a common followup topic (generators)